docs/installation/linux.md

```diff
@@ -640,50 +636,6 @@ and remove the desired the paths with `sudo rm`.
 
 [paths]:#paths
 
-### OpenSSL on macOS
```

Comment:
Should this be moved somewhere else, or does it just not work?

Reply:
As I mentioned in the PR description, It's not applicable anymore. See the announcement [".NET Core 2.0 Cryptography uses Apple Security Framework on macOS"](https://github.com/dotnet/announcements/issues/21) and the [corresponding PR](https://github.com/dotnet/corefx/pull/17011)